charts/hawkbit/templates/ingress.yaml

```diff
@@ -1,6 +1,10 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "hawkbit.fullname" . -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
i would remove this because helm 3 likely does not work with it at the moment.
see: https://github.com/helm/helm/issues/7004